src/plotly.js

```diff
@@ -47,6 +47,7 @@ exports.Colorscale = require('./components/colorscale');
 exports.Colorbar = require('./components/colorbar');
 exports.ErrorBars = require('./components/errorbars');
 exports.Annotations = require('./components/annotations');
+exports.RangeSlider = require('./components/rangeslider');
```

Comment:
Oop. Artifact from a commit where I called a `RangeSlider.setRange` method. I'll remove this.